src/parquet/column_reader.cc

```diff
@@ -182,7 +182,10 @@ std::shared_ptr<Page> SerializedPageReader::NextPage() {
     // Read the compressed data page.
     buffer = stream_->Read(compressed_len, &bytes_read);
     if (bytes_read != compressed_len) {
-      ParquetException::EofException();
+      std::stringstream ss;
+      ss << "Page was smaller (" << bytes_read << ") than expected (" << compressed_len
+         << ")";
+      ParquetException::EofException("Page was smaller than expected");
```

Comment:
Shouldn't this return:
ParquetException::EofException(ss.str());
?

Reply:
yes, it should. I'll fix this now.